linux57-tkg/install.sh

```diff
@@ -44,17 +47,46 @@ _misc_adds="false" # We currently don't want this enabled on non-Arch
 
 if [ "$1" = "install" ] || [ "$1" = "config" ]; then
 
-  source linux*-tkg-config/prepare
+  if [ -z $_distro ]; then
+    
+    echo "Which linux distribution are you running ?"
+    echo "if it's not on the list, chose the closest one to it: Fedora/Suse for RPM, Ubuntu/Debian for DEB"
+    echo "   1) Debian"
+    echo "   2) Fedora"
+    echo "   3) Suse"
+    echo "   4) Ubuntu"
+    read -p "[1-4]: " _distro_index
+
+    if [ "$_distro_index" = "1" ]; then
+      _distro="Debian"
+    elif [ "$_distro_index" = "2" ]; then
+      _distro="Fedora"
+    elif [ "$_distro_index" = "3" ]; then
+      _distro="Suse"
+    elif [ "$_distro_index" = "4" ]; then
+      _distro="Ubuntu"
+    else
+      echo "Wrong index. aborting..."
+      exit 0
```

Comment:
Rather than bailing out, this should prompt again for the distribution, like [here](https://github.com/Frogging-Family/linux-tkg/blob/9c3c155b3ed07a4aa04ec10d6fc0e8b5710a2f07/linux57-tkg/linux57-tkg-config/prepare#L824-L836).

Reply:
Laziness had struck me there xD Okay will do!